lib/AST/ASTStructuralEquivalence.cpp

```diff
@@ -338,6 +430,23 @@ static bool IsStructurallyEquivalent(StructuralEquivalenceContext &Context,
   llvm_unreachable("Invalid template argument kind");
 }
 
+/// Determine whether two template argument lists are equivalent.
```

Comment:
Is this hunk related?

Reply:
Ok, I see you use it when comparing the contexts.

Reply:
The function is used in `IsEquivalentContext`.